trading_calendars/exchange_calendar_six.py

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import time
+from pandas.tseries.holiday import (
+    Easter,
+    EasterMonday,
+    GoodFriday,
+    Holiday,
+)
+from pandas.tseries.offsets import Day
+from pytz import timezone
+
+from .trading_calendar import (
+    TradingCalendar,
+    HolidayCalendar,
+)
+
+# Regular Holidays
+# ----------------
+SIXNewYearsDay = Holiday(
+    "New Year's Day",
+    month=1,
+    day=1,
+)
+
+BerchtoldsDay = Holiday(
+    "Berchtold's Day",
+    month=1,
+    day=2,
+)
+
+LabourDay = Holiday(
+    "Labour Day",
+    month=5,
+    day=1,
+)
+
+AscensionDay = Holiday(
+    "Ascension Day",
+    month=1,
+    day=1,
+    offset=[Easter(), Day(39)],
+)
+
+WhitMonday = Holiday(
```

Comment:
I notice this is duplicated with the definition in the German calendar, except that in germany they didn't observe this until 2010. Maybe we should add a `common_holidays.py` or something and put a helper function there like:

```
def whit_monday(observance):
    return Holiday("Whit Monday", ..., observance=observance)
```

cc @dmichalowicz this might also make sense for the calendars you're looking at.

Reply:
Yup, I'd thought about using the FWB's Whit Monday, and didn't because of the start date issue. Will move this and the Christmas-related holidays to that file